public/app/plugins/panel/news/feed.ts

```diff
@@ -9,8 +9,18 @@ export async function fetchFeedText(url: string) {
 
 export function isAtomFeed(txt: string) {
   const domParser = new DOMParser();
-  const doc = domParser.parseFromString(txt, 'text/xml');
-  return doc.querySelector('feed') !== null;
+
+  //@ts-ignore
+  if (trustedTypes.createPolicy) {
+    //@ts-ignore
```

Comment:
I know this is a draft, just flagging for later that we absolutely cannot accept `//@ts-ignore` into the code.

Reply:
@joshhunt In https://github.com/grafana/grafana/pull/64975/commits/eb317271f6d5bb51dde19532822849a2bbcbcb12 I managed to get it working without `//@ts-ignore` by using type assertions. Even though that should be avoided, it works really good. I do see other places where we use `as unknown as string` so my question is if using type assertions is fine here?